### DISCUSSION TITLE
两年续约已交费 但未收到90天的赠送时长

### DISCUSSION THREAD

**Discussion by seefunha** [getlantern/lantern-forum-cn#711] [General]
![微信图片_20240710145959](https://github.com/getlantern/lantern-forum-cn/assets/175191413/be56aaa1-b3e1-459b-b4b8-71c0a276fe6f)
账号924644509@qq.com

**Comment by MaxMKMax** [comment]
@seefunha 您好. 我已经帮您提交问题了. 相关部门会给您加时间. 谢谢您支持蓝灯!

**Reply by seefunha** [reply]
你好，请在帮我追下这个问题。截至目前并未解决，我是7/9交的费，但目前有效期是到08/23/2026，少了45天

**Reply by MaxMKMax** [reply]
@seefunha 嗯嗯. 好的, 我帮您问一下. 您方便也提供我一下支付单据吗?

**Reply by seefunha** [reply]
![微信图片_20240723153046](https://github.com/user-attachments/assets/c5f2b137-e406-44ae-a777-747d2694cb5a)
![Uploading 微信图片_20240723153037.jpg…]()

**Comment by seefunha** [comment]
请问解决了吗



在 2024-07-23 00:37:14，"Mike K" ***@***.***> 写道：

@seefunha 嗯嗯. 好的, 我帮您问一下. 您方便也提供我一下支付单据吗?

**Reply by MaxMKMax** [reply]
@seefunha 好的. 没问题. 我再帮您追问一下. 谢谢您支持蓝灯.

**Reply by MaxMKMax** [reply]
@seefunha 您好. 应该帮您加了时间了. 请查看.

**Reply by seefunha** [reply]
![6259adb0754eda97ac1d23486844c42b](https://github.com/user-attachments/assets/bdfb3faa-88f3-4e65-9b8d-8da0c50032a0)
还没变化 请再确认 麻烦了

**Reply by seefunha** [reply]
> @seefunha 您好. 应该帮您加了时间了. 请查看.

好了 感谢
